.changes/v3.6.0/782-improvements.md

```diff
@@ -0,0 +1 @@
+* `data/vcd_storage_profile` add IOPS settings and other data to state [GH-782]
```

Comment:
Please list specific fields that were added - someone may benefit of that.

Reply:
added

Reply:
Thanks. I want that message to be concrete though: is there a reason to have the two words "like", can they be removed?

Reply:
changed